maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -45,16 +45,26 @@
       <artifactId>maven-scm-provider-git-commons</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.eclipse.jgit</groupId>
-      <artifactId>org.eclipse.jgit</artifactId>
-      <version>4.5.4.201711221230-r</version>
+      <artifactId>plexus-interactivity-api</artifactId>
+      <groupId>org.codehaus.plexus</groupId>
+      <version>1.1</version>
       <exclusions>
         <exclusion>
-          <groupId>commons-logging</groupId>
-          <artifactId>commons-logging</artifactId>
+          <groupId>org.codehaus.plexus</groupId>
+          <artifactId>plexus-container-default</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jgit</groupId>
+      <artifactId>org.eclipse.jgit</artifactId>
+      <version>5.13.1.202206130422-r</version><!-- since version 6 requires Java 11 -->
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jgit</groupId>
+      <artifactId>org.eclipse.jgit.ssh.apache</artifactId>
+      <version>5.13.1.202206130422-r</version><!-- since version 6 requires Java 11 -->
+    </dependency>
```

Comment:
Is this strictly required to test gitexe ssh alternative private key? I don't think so. If not, I'd recomment to remove the JGit bits from here, complete gitexe with whatever is required then do two JGit PRs. WDYT?